src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -113,24 +113,34 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
             if (input.contents) {
                 // Get the relationships
                 input.contents.forEach((content) => {
+                    const foundComponentTarget = models.find(
+                        (model) => model['@id'] === content.schema
+                    );
```

Comment:
We should probably only do this if we're dealing with a component? I.e. under the `case` statement for component.